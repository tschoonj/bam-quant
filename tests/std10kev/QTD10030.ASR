File: C:\xrfdaten\2014KW30\chris\std10kev\std10030.spe
Start: 21.07.2014 16:08:21
Stop: 21.07.2014 16:10:01
BESSY: 298.3435923 298.3271653
IONI: 1.419501E-10 1.414584E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 63724
Run: 100000
Dead: 36
ICR: 135160
OCR: 0
Accumulated BESSY: 59710.9809847
Accumulated Ioni: 0.000000028382395
N Accumulation: 200
Normfactor:  1.1058034e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
exp_x 132.020401000977 132.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  30   1      8.631   7555850.      2754.     115.56
