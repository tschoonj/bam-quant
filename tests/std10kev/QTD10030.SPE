$DATA:
0 2000
14747
9878
4566
1446
309
71
15
12
13
7
4
0
0
1
2
6
141
1416
2092
2157
2038
1782
1800
1700
1733
1653
1553
1605
1420
1558
1396
1312
1277
1345
1213
1272
1178
1234
1165
1147
1139
1159
1103
1081
1118
1059
1001
1060
997
953
937
951
967
991
940
927
965
988
957
940
937
891
1000
910
946
917
909
855
917
883
877
825
897
828
877
832
836
826
822
807
804
848
825
750
814
793
762
783
822
731
817
792
760
781
798
836
810
808
823
843
865
849
873
840
833
800
830
788
788
776
777
738
767
742
740
669
762
732
734
724
706
637
668
761
730
755
717
781
749
854
760
780
787
672
735
770
696
673
702
738
711
766
789
868
915
1067
1122
1100
990
908
852
731
681
713
700
692
702
683
725
728
668
651
605
628
623
599
630
641
660
621
670
673
681
635
587
635
653
563
638
620
626
627
629
694
620
664
622
628
616
631
610
656
600
625
626
656
621
607
650
638
658
624
635
625
669
645
661
612
590
665
668
582
612
643
630
670
636
634
644
591
595
630
658
663
626
647
642
621
629
589
602
643
632
695
639
631
657
615
626
633
641
634
651
622
646
624
697
632
668
655
638
644
644
620
597
621
692
633
657
625
620
674
611
666
655
642
680
676
636
698
713
678
683
734
712
669
695
698
671
716
685
710
739
705
663
717
733
676
731
787
667
658
760
721
754
677
758
747
793
721
744
713
802
770
735
787
768
787
793
818
804
821
871
924
979
1056
1096
1041
1075
1060
1035
934
966
943
901
859
868
869
904
921
930
978
1008
1046
1179
1302
1513
1754
2026
2338
2386
2523
2363
2180
1907
1704
1455
1364
1195
1185
1158
1215
1094
1129
1114
1187
1191
1223
1206
1170
1239
1222
1230
1241
1306
1320
1335
1253
1371
1294
1287
1370
1335
1275
1334
1355
1331
1370
1328
1286
1379
1376
1516
1473
1570
1669
1647
1696
1798
1748
1854
1772
1829
1850
1865
1951
2004
2166
2154
2215
2390
2472
2598
2645
2853
3064
3168
3457
3555
3804
4248
4536
4957
5729
6556
8370
11394
18887
32706
59090
104914
176990
280131
412602
558393
702407
811128
862297
844710
760860
629289
480955
338393
219392
131655
73780
38803
19780
10152
5602
3510
2571
2083
1828
1696
1585
1562
1490
1505
1484
1505
1555
1591
1695
1806
1915
2075
2260
2492
2645
2931
3306
3926
5027
6871
10143
15398
23914
35695
51314
70100
88711
104927
115191
117594
111505
96928
79124
59143
41598
27090
16586
9416
5224
2722
1410
791
533
409
394
369
357
415
368
330
364
313
306
256
245
181
158
127
116
81
55
64
46
43
48
47
45
42
42
49
48
39
43
40
31
39
34
47
39
21
38
29
32
37
33
41
38
34
29
40
32
25
21
33
21
21
33
35
34
26
27
27
22
24
25
23
27
22
23
25
25
18
23
23
30
23
23
23
23
20
20
23
24
17
27
26
33
23
24
31
20
23
28
19
23
16
18
16
20
25
22
16
26
19
23
22
18
10
17
19
12
11
22
12
19
21
15
14
16
17
18
14
27
12
15
19
25
14
19
19
20
11
17
19
17
19
17
17
23
22
17
15
18
11
20
23
20
19
19
16
11
8
23
26
17
16
12
22
13
19
12
11
10
18
16
14
15
13
20
19
15
14
13
18
17
16
11
20
14
17
11
16
27
15
21
10
17
15
18
22
13
12
18
21
20
17
14
15
10
23
22
15
24
10
21
11
16
11
16
10
14
17
20
16
19
22
8
19
16
12
15
14
12
25
11
12
19
19
16
15
11
16
11
26
18
12
19
17
19
15
10
16
16
16
17
12
19
17
17
16
17
20
26
15
21
20
20
18
16
23
27
30
30
18
28
27
21
27
24
19
20
25
25
26
31
32
32
37
38
54
39
31
35
43
41
43
54
34
30
31
23
23
27
30
28
25
29
29
29
28
29
36
28
26
32
29
20
23
24
34
35
27
24
27
31
28
24
20
32
17
37
22
30
34
33
29
29
23
36
29
42
38
32
34
26
44
49
44
40
35
47
36
59
64
57
54
56
64
86
68
92
94
115
137
169
271
339
507
746
1041
1453
2061
2700
3583
4391
5199
5797
6467
6913
6816
6680
6078
5312
4615
3878
3018
2196
1576
1119
736
497
288
205
110
85
49
60
46
39
38
36
45
48
41
46
48
58
59
63
71
89
103
130
160
217
309
439
566
757
921
1190
1403
1539
1816
1853
1855
1873
1711
1599
1385
1167
886
687
541
419
256
190
104
75
59
32
21
25
12
20
8
7
17
14
12
13
4
8
11
7
13
10
13
15
16
17
20
30
45
58
65
77
87
88
118
109
152
136
132
109
86
97
77
65
45
31
28
15
12
6
6
5
2
1
1
0
1
0
2
3
0
0
1
0
0
0
0
0
0
0
1
0
1
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
2
1
0
1
0
0
0
0
0
1
0
0
0
0
0
3
0
1
0
1
0
0
0
1
1
0
1
0
0
0
0
0
2
0
0
1
0
0
1
0
0
0
0
0
0
0
1
0
1
1
1
0
1
0
0
0
0
0
1
1
0
0
0
1
1
0
0
1
0
1
0
0
0
0
2
2
0
1
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
2
0
0
0
0
1
0
1
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
0
1
0
0
0
0
0
0
0
0
0
0
1
1
0
1
0
1
1
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
1
1
0
1
0
0
1
3
2
1
0
3
1
1
5
1
5
7
8
16
18
19
21
22
27
34
38
36
36
31
30
35
34
30
34
19
23
12
12
5
6
3
4
1
0
2
0
1
0
0
0
1
0
0
0
0
1
2
0
0
2
2
4
4
5
4
11
12
17
5
5
25
10
15
15
19
9
14
14
19
11
12
12
6
8
5
4
4
3
1
1
0
1
1
0
0
0
0
1
0
0
0
1
0
0
0
0
0
1
0
0
1
0
1
0
2
2
2
2
1
5
4
1
1
1
2
2
2
0
0
1
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10030.spe
Start: 21.07.2014 16:08:21
Stop: 21.07.2014 16:10:01
BESSY: 298.3435923 298.3271653
IONI: 1.419501E-10 1.414584E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 63724
Run: 100000
Dead: 36
ICR: 135160
OCR: 0
Accumulated BESSY: 59710.9809847
Accumulated Ioni: 0.000000028382395
N Accumulation: 200
Normfactor:  1.1058034e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
 
exp_x 132.020401000977 132.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  30   1      8.631   7555850.      2754.     115.56
