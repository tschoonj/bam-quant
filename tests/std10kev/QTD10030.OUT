     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:06  บ   
     บ  Spectrum: QTD10030.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   231.3   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -47.0 ๑   100.0   บ     -47.0 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.266 ๑   2.000   บ    20.266 ๑    .002   บ   
     บ det NOISE (eV) บ     -101.2 ๑    40.0   บ    -101.2 ๑    -3.2   บ   
     บ  FANO factor   บ       .206 ๑    .050   บ      .206 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Zn-Ka                           7555850. ๑   2754.           บ   
     บ    KA1         8.639    .66048     4990459. ๑   1824.    115.6  บ   
     บ              428.596    166.78       17762.           9.65E-01  บ   
     บ    KA2         8.616    .33952     2565391. ๑    945.    106.7  บ   
     บ              427.461    166.48       17837.           9.65E-01  บ   
     บ                                                                 บ   
     บ  2 Zn-Kb                           1079692. ๑   1047.           บ   
     บ    KB1         9.572   1.00000     1079692. ๑   1053.     61.7  บ   
     บ              474.634    178.68       11823.           9.74E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
